src/main/java/core/basesyntax/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetArea {
+    public abstract double getArea();
```

Comment:
Interface methods are by default abstract
```suggestion
    public double getArea();
```